hagrid/api/serializers.py

```diff
@@ -0,0 +1,28 @@
+from hagrid.products.models import Variation, Product, Size, SizeGroup
+from rest_framework import serializers, viewsets
+
+
+class ProductAPISerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Product
+        fields = '__all__'
+
+
+class SizeGroupAPISerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SizeGroup
+        fields = '__all__'
+
+
+class SizeAPISerializer(serializers.ModelSerializer):
+    group = SizeGroupAPISerializer()
+    class Meta:
+        model = Size
+        fields = '__all__'
+
+
+class VariationAPISerializer(serializers.ModelSerializer):
+    size = SizeAPISerializer()
```

Comment:
Do you think we should also include `product = ProductAPISerializer()`, so a single call to this endpoint is enough for most applications?

Reply:
Sure, added